<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_rec</class>
 <widget class="QDialog" name="Dialog_rec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aufnahme / Protokollierung</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Aufnahmedatei</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="Btn_open">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ã–ffnen</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_save">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Speichern</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>194</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Zeitabstand</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dSpinBx_dt">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Steuerbefehle</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lEd_send"/>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_send">
          <property name="text">
           <string>Senden</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="Btn_rec">
          <property name="text">
           <string>Aufzeichnen</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_play">
          <property name="text">
           <string>Wiedergeben</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBx_wdh">
        <property name="text">
         <string>wiederholt abspielen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txtEd_rec">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Btn_clear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_close">
       <property name="text">
        <string>Verstecken</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lEd_send</tabstop>
  <tabstop>Btn_send</tabstop>
  <tabstop>Btn_rec</tabstop>
  <tabstop>Btn_play</tabstop>
  <tabstop>checkBx_wdh</tabstop>
  <tabstop>txtEd_rec</tabstop>
  <tabstop>Btn_open</tabstop>
  <tabstop>Btn_save</tabstop>
  <tabstop>dSpinBx_dt</tabstop>
  <tabstop>Btn_clear</tabstop>
  <tabstop>Btn_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_rec</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lEd_send</sender>
   <signal>editingFinished()</signal>
   <receiver>Btn_send</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>txtEd_rec</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
